### PR TITLE
Increases el8 root volume size to address yum package failures

### PR DESCRIPTION
**Pull request by lorengordon**

### CODE REVIEW

**Comment by lorengordon** [comment]
go codebuild go

File: spel/minimal-linux.pkr.hcl [hunk 1]
```diff
@@ -539,7 +539,15 @@ variable "amigen8_source_url" {
 variable "amigen8_storage_layout" {
   description = "List of colon-separated tuples (mount:name:size) that describe the desired partitions for LVM-partitioned disks on EL8 images"
   type        = list(string)
-  default     = []
+  default = [
+    "/:rootVol:5",
```
Comment (ferricoxide) [inline spel/minimal-linux.pkr.hcl:543]: Only increasing the root LVM volume by 1GiB? Is that going to be sufficient beyond the near-term?

File: spel/README.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ No resources.
...
 | <a name="input_amigen8_source_branch"></a> [amigen8\_source\_branch](#input\_amigen8\_source\_branch) | Branch that will be checked out when cloning AMIgen8 | `string` | `"master"` | no |
 | <a name="input_amigen8_source_url"></a> [amigen8\_source\_url](#input\_amigen8\_source\_url) | URL that will be used to clone AMIgen8 | `string` | `"https://github.com/plus3it/AMIgen8.git"` | no |
-| <a name="input_amigen8_storage_layout"></a> [amigen8\_storage\_layout](#input\_amigen8\_storage\_layout) | List of colon-separated tuples (mount:name:size) that describe the desired partitions for LVM-partitioned disks on EL8 images | `list(string)` | `[]` | no |
+| <a name="input_amigen8_storage_layout"></a> [amigen8\_storage\_layout](#input\_amigen8\_storage\_layout) | List of colon-separated tuples (mount:name:size) that describe the desired partitions for LVM-partitioned disks on EL8 images | `list(string)` | <pre>[<br>  "/:rootVol:5",<br>  "swap:swapVol:2",<br>  "/home:homeVol:1",<br>  "/var:varVol:2",<br>  "/var/tmp:varTmpVol:2",<br>  "/var/log:logVol:2",<br>  "/var/log/audit:auditVol:100%FREE"<br>]</pre> | no |
```
Comment (ferricoxide) [inline spel/README.md:29]: See my comment/question on the `spel/minimal-linux.pkr.hcl`  file.

**Review by ferricoxide — APPROVED** [review]
If people bitch inside the first few months of the AMIs getting used, we can adjust fire _then_.

**Pull request merged**